rust-toolchain

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.62.0"
+channel = "1.67.0"
```

Comment:
we agree with @juanbono  to do it because of annoying errors that were appearing on Vscode, beyond that it is not actually necessary but it does not break anything either and also allows to use more actual features.

Reply:
Is it related to procedural macros?